client/room.html

```diff
@@ -8,7 +8,7 @@
   <link rel="icon" type="image/svg+xml" href="/i/branding/favicon.svg"/>
   <link rel="alternate icon" href="/i/branding/favicon.png"/>
   <link rel="preconnect" href="https://fonts.gstatic.com">
-  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link href="/i/fonts/fonts.css" rel="stylesheet">
   <script src="https://cdn.jsdelivr.net/combine/npm/vue@3.0.7/dist/vue.global.prod.js,npm/jszip@3.6.0/dist/jszip.min.js"></script>
   <base target="_blank">
   <style> {{CSS}} </style>
```

Comment:
Any reason why you link the new `fonts.css` directly instead of adding it to `{{CSS}}` through `server/minify.mjs`?

Reply:
Mostly because I did not understand how that worked or was even there. I replaced the previous line with one that had an identical function. I can add it to the minify, though. 

Minify makes things hard to debug when things do go wrong. (The error is on line 1? What?!)

Reply:
It won't be minified when launched with `NOCOMPRESS=1` like `npm run debug` does. The PR test servers run like that.

Reply:
Neat! 

I just committed and pushed an update with the file in minify.